fractions/jaeger/src/main/java/org/wildfly/swarm/jaeger/JaegerFraction.java

```diff
@@ -0,0 +1,84 @@
+package org.wildfly.swarm.jaeger;
+
+import org.jboss.logging.Logger;
+import org.wildfly.swarm.spi.api.Fraction;
+import org.wildfly.swarm.spi.api.annotations.Configurable;
+import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
+
+import static com.uber.jaeger.Configuration.*;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@DeploymentModule(name = "org.wildfly.swarm.jaeger", slot = "deployment")
```

Comment:
> With the above line, you're adding that classes in your *.deployment package to every deployment. Which might be what you want, I dunno.

Perhaps if I explain the goal, it might be clearer. The main goal is to provide the developer with access to a proper combination of Jaeger+OpenTracing libraries. 

I tried to extend the OpenTracing fraction, to just add the Jaeger dependencies to the deployment, but couldn't get that working. The OpenTracing fraction has a `TracerResolver`, which tries to load a service via the `ServiceLoader` facility. The Jaeger fraction has a provider for this service, but it looks like the OpenTracing fraction can't "see" this provider class (they are possibly in different classloaders?). If a `Tracer` is resolved by the `TracerResolver`, it is then fed into the `GlobalTracer`, which can be consumed by the target application to create "business spans".

So, the final solution is to just "depend" on the OpenTracing fraction and implement a flag, set by the Jaeger fraction, telling the OpenTracing to not try to resolve the tracer via the `TraceResolver`. This way, the Jaeger fraction will register the tracer by itself with the `GlobalTracer`. One advantage of this is that it allows the target application to configure the Fraction like other Swarm fractions, the disadvantage is that this is far more complex than just adding the Jaeger dependencies to the target WAR.